src/tree/GroupTreeItemBase.ts

```diff
@@ -32,7 +34,8 @@ export class GroupTreeItemBase extends AzExtParentTreeItem {
     }
 
     public get contextValue(): string {
-        return this.config.contextValue || 'GroupTreeItem';
+        const contextValues = new Set([...this.config.contextValuesToAdd ?? [], ...this.internalContextValuesToAdd, 'group']);
```

Comment:
It makes it easier to implement focus group command since we know `group` will always be there. Instead of relying on having the config context value follow the `*Group` pattern.